2-devtips.md

```diff
@@ -13,6 +13,14 @@
   - Select left/right: Ctrl+shift+arrow
   - Box select: alt
 
+> **Exercise**: Explore code navigation
+>
+> 1. Create a new folder called "tutorial" and a new file "hello.py" with a function `say_hello` that prints a message using the `emoji` library.
+> 2. Create a new function `my_zen_of_python` that prints out the text of Zen of Python.
+> 3. Replace the word "is" with the word "was". How many keystrokes does it take to make this replacement in the editor?
```

Comment:
this is in the source code?

do people even need to have a word "is" in the source code

Reply:
I thought it would be fun to edit text but we can do code instead.

Reply:
I've added a line to output the text to a markdown file.